index.js

```diff
@@ -39,35 +40,23 @@ module.exports = function (content) {
     }
 
     var filePath = prefix + resource.slice(relativeToIndex + relativeTo.length); // get the base path
-    var html;
+    var html, match = content.match(/^module\.exports.+?(".*")/);
 
-    if (content.match(/^module\.exports/)) {
-        var firstQuote = findQuote(content, false);
-        var secondQuote = findQuote(content, true);
-        html = content.substr(firstQuote, secondQuote - firstQuote + 1);
+    // if match, html is the captured match
+    if (match) {
+        html = match[1]
     } else {
         html = content;
```

Comment:
and i just find that `console.log(require('../../index.js!./test.html'));` will crash webpack. So what is **else** used for?